common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -152,4 +152,17 @@ public static void createBlockPath(String path) throws IOException {
   public static void move(File from, File to) throws IOException {
     Files.move(from, to);
   }
+
+  /**
+   * Delete the file or directory
+   *
+   * Current implementation uses {@link java.io.File#delete();}, may change if
+   * there is a better solution.
+   *
+   * @param file file to delete
+   * @throws IOException when fails to delete
+   */
+  public static boolean delete(File file) throws IOException {
+    return file.delete();
```

Comment:
The javadoc is inconsistent with how the method behaves. When it failed to delete this file, this method returns `false` rather than throwing an exception. Also `file.delete()` actually not throwing `IOException` but `SecurityException`. 

Here is my suggestion: I think it is fine to keep the signature to throw `IOException`, but changing the returned type from `boolean` to `void` and if `file.delete()` fails, throw `IOException`? That is more aligned to java nio and in the long run, we will re-implement this method using java nio package, the reason we are still not doing so is because nio is not included in java6 and Tachyon is still supporting java6.